package.json

```diff
@@ -12,7 +12,7 @@
   ],
   "private": false,
   "version": "6.3.0",
-  "branch": "6.x",
+  "branch": "6.3",
```

Comment:
Is this just changed for testing purposes?

Reply:
I think this was missed when the 6.3 branch was cut the other day

Reply:
Ah, okay!